target/linux/ath79/image/generic.mk

```diff
@@ -616,3 +624,21 @@ define Device/yuncore_a770
   IMAGE_SIZE := 16000k
 endef
 TARGET_DEVICES += yuncore_a770
+
+define Device/yuncore_cpe830
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := CPE830
+  DEVICE_PACKAGES := uboot-envtools kmod-ath9k kmod-i2c-core kmod-i2c-gpio
+endef
+TARGET_DEVICES += yuncore_cpe830
+
+define Device/zbtlink_zbt-wd323
```

Comment:
I can't see this board in your PR, so it should probably be dropped?

Reply:
Actually, my fork of openwrt on github is quite old, soit creates problems in the PR. Not sure how to fix it.

I would rather have my other PR be merged into master, so I recreate one for this one only

Reply:
Actually, my fork of openwrt on github is quite old, soit creates problems in the PR. Not sure how to fix it.

I would rather have my other PR be merged into master, so I recreate one for this one only

Reply:
I removd the zbtlink paragraphe